spring-boot-admin-server-ui/src/main/frontend/components/sba-action-button-scoped.vue

```diff
@@ -121,6 +123,9 @@ export default {
         )
         .subscribe();
     },
+    emitScopeChanged(scope) {
+      this.$emit('scopeChanged', scope);
+    },
```

Comment:
If we already have this button and use it in other places, why do we need to add this event here?

@SteKoe is this the way to go or should something else be used for this feature?

Reply:
I'm not a frontend developer, but i think we need to propagate somehow actual scope. From what i've read in Vue docs - for `child -> parent` propagation we need to emit event with the change. Event is emitted, but other sba-action-button-scoped consumers aren't forced to consume, because it's optional.

Reply:
After having a deeper look into the change, I guess this is a good approach to propagate the scope of the action button to surrounding components. It feels a little bit confusing, as I can now switch the scope for the entire page via one button at the top of the page, but this was requested ind #2181. So, I will merge this as is, thanks for your work and patience :)